src/App.js

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
-//import logo from './logo.svg';
+import logo from './logo.svg';
 //import './App.css';
 import {Route} from 'react-router-dom';
 import 'bootstrap/dist/css/bootstrap.min.css';
 
 import {Root} from './components/Root';
 import {Home} from './components/Home';
 import Test from './components/TestPage.js';
+import {Mode} from './components/Mode.js';
 
 
 class App extends React.Component {
   render() {
     return (
-      <div>
+      <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <Mode />
+      </header>
+      {/* <div>
```

Comment:
Why are you removing this?  The routes are made so that we can have multiple pages, such as the home page and the test page.  We already had a designated home page.  Just put your mode component in the Home component